docs/tutorials/nominate/vertex_nomination_via_SGM.ipynb

```diff
@@ -0,0 +1,166 @@
+{
+ "cells": [
+  {
```

Comment:
I think the tutorial needs more info about the algorithm, intuition behind it, and why it's useful. The ASE tutorial is a useful template I think